server/src/Packages/Stripe/Actions/StripeBeforeCheckout.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Gernzy\Server\Packages\Stripe\Actions;
+
+use \App;
+use Gernzy\Server\Classes\ActionClass;
+use Gernzy\Server\Exceptions\GernzyException;
+use Gernzy\Server\Services\ActionInterface;
+use Gernzy\Server\Services\ExhangeRatesManager;
+
+class StripeBeforeCheckout implements ActionInterface
+{
+    public function __construct()
+    {
+    }
+
+    public function run(ActionClass $action)
+    {
+        // Get the data passed on from the event
+        $data = $action->getOriginalData();
+
```

Comment:
Somewhere in here, you'll check for that provider string and return immediately if it does not match

Reply:
@luke-siedle I implemented this